src/main/java/models/ArticleModel.java

```diff
@@ -17,27 +17,38 @@
         if (con != null) {
             try {
 
-
                 Statement statement = con.createStatement();
                 statement.setQueryTimeout(30);  // set timeout to 30 sec.
 
-                String sql = "SELECT BodyHtml FROM Examples WHERE DocTopicId = ?";
+                String sql_title = "SELECT Title FROM Topics WHERE Id = ?";
                 PreparedStatement ps;
-                ps = con.prepareStatement(sql);
+                ps = con.prepareStatement(sql_title);
                 ps.setInt(1, selectedArticle.getId());
+                ResultSet rs;
+                rs = ps.executeQuery();
 
+                String title = null;
+
+                while (rs.next()) {
+                    title = rs.getString("Title");
+                }
+
+                String sql_example = "SELECT BodyHtml FROM Examples WHERE DocTopicId = ?";
```

Comment:
It would be better do not leave unused code

Reply:
Can't understand which code is unused?

Reply:
Here are two quesries to get examples and title of the article. Both fields are added to object

Reply:
Ok